connexion/apis/aiohttp_api.py

```diff
@@ -316,66 +316,57 @@ def get_response(cls, response, mimetype=None, request=None):
 
         url = str(request.url) if request else ''
 
-        logger.debug('Getting data and status code',
-                     extra={
-                         'data': response,
-                         'url': url
-                     })
-
-        if isinstance(response, ConnexionResponse):
-            response = cls._get_aiohttp_response_from_connexion(response, mimetype)
+        response = cls._get_response(response, mimetype=mimetype, url=url)
 
-        if isinstance(response, web.StreamResponse):
-            logger.debug('Got stream response with status code (%d)',
-                         response.status, extra={'url': url})
-        else:
-            logger.debug('Got data and status code (%d)',
-                         response.status, extra={'data': response.body, 'url': url})
+        # TODO: I let this log here for full compatibility. But if we can modify it, it can go to _get_response()
+        logger.debug('Got stream response with status code (%d)',
+                     response.status, extra={'url': url})
```

Comment:
Why would we need to retain a log line for compatibility? Would we ever consider a debug-level log entry as "API"?

Reply:
Ah. At this point the response is a framework response. So if we want to log the status code, then it needs to be under the framework specific *_api.py

Reply:
resolved